_data/users.yml

```diff
@@ -197,6 +197,17 @@
 - name: Myntra
   logo: /assets/images/logos/myntra.png
   logosmall: /assets/images/logos/myntra-small.png
+  testimonial: |
```

Comment:
Add an `anchor:` so that clicking on their logo takes you to the testimonial

Reply:
correct.. add after name .. see others like linkedin

Reply:
added lol realized this after I pushed